docs/source/authentication.rst

```diff
@@ -1,183 +1,19 @@
-==============
-Authentication
-==============
+=====================
+Client authentication
+=====================
 
-When connecting to Mesh using the Python SDK you might need to authenticate
-using Kerberos. This is required by the Mesh server if Kerberos is enabled
-for the gRPC interface in the Mesh configuration file.
+Mesh server could be configured to protect its gRPC API so that only
+authenticated users that are assigned to specific roles can access it. There
+are two options:
```

Comment:
```suggestion
A Mesh server may be configured to protect its gRPC API so that only
authenticated users that are assigned to specific roles can access it. There
are two options:
```